terrain_navigation_ros/launch/mavros.launch.py

```diff
@@ -1,22 +1,47 @@
 import os
 
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import TextSubstitution
 from ament_index_python.packages import get_package_share_directory
 from launch import LaunchDescription
+from launch.conditions.launch_configuration_equals import LaunchConfigurationEquals
 from launch_ros.actions import Node
 
 
+FLIGHT_STACK_PX4 = "px4"
+FLIGHT_STACK_ARDUPILOT = "ardupilot"
+FLIGHT_STACK_ARG_NAME = "flight_stack"
```

Comment:
Could we default to px4 for now, since this is the only flight stack that is functionally supported?

Reply:
Yup